### DISCUSSION TITLE
Any new lyric sources you can use

### DISCUSSION THREAD

**Discussion by SolarAquarion** [osdlyrics/osdlyrics#108] [Ideas]

**Comment by SolarAquarion** [comment]
@PedroHLC Genius won't work?

**Comment by PedroHLC** [comment]
Doesn't seem genius expose lyrics through their API (at least is not in docs.genius.com, and an example they mention don't have any code at all); similar service MusiXMatch requires payment for the token and don't use LRC (use a XML-based format instead), probably Genius will follow the same path.

**Comment by SolarAquarion** [comment]
@PedroHLC https://happi.dev/docs/music it seems this can work also this works from Genius, and it's free, I think https://github.com/johnwmillr/LyricsGenius

**Comment by SolarAquarion** [comment]
@PedroHLC EvilLyrics? http://www.evillabs.sk/evillyrics/

**Comment by PedroHLC** [comment]
That's better, it uses a timestamped format, but it's not our conventional LRC. I'll do a small planning of what would be needed for that to be implemented...

**Comment by SolarAquarion** [comment]
I was inspired by this script https://bazaar.launchpad.net/~lrcshow-x/lrcshow-x/trunk/view/head:/engines/engine_evillyrics.py

**Comment by SolarAquarion** [comment]
I wonder if Baidu, qq, and whomstever can still be used https://bazaar.launchpad.net/~lrcshow-x/lrcshow-x/trunk/files/94/engines

**Comment by PedroHLC** [comment]
Yep, both API's are not so different, take a look at:
https://github.com/osdlyrics/osdlyrics/tree/master/lyricsources
**EDIT**: Do not that for Evil Lyrics, lyrics are converted from KAR to LRC while downloading

**Comment by SolarAquarion** [comment]
https://syair.info/ https://gitlab.com/ronie/script.cu.lrclyrics/-/blob/master/lib/culrcscrapers/syair/lyricsScraper.py

ttplayer https://gitlab.com/ronie/script.cu.lrclyrics/-/blob/master/lib/culrcscrapers/ttplayer/lyricsScraper.py

minilyrics https://gitlab.com/ronie/script.cu.lrclyrics/-/blob/master/lib/culrcscrapers/minilyrics/lyricsScraper.py

GomAudio https://gitlab.com/ronie/script.cu.lrclyrics/-/blob/master/lib/culrcscrapers/gomaudio/lyricsScraper.py

**Comment by cirrusUK** [comment]
i recently installed lyrico with pip in virtualenv, it uses 
    http://lyrics.wikia.com/wiki 
    https://www.musixmatch.com/lyrics/
    https://www.lyricsmode.com/lyrics

and finds a suprisingly good amount of lyrics

**Comment by eadmaster** [comment]
> i recently installed lyrico with pip in virtualenv, it uses

[only supports unsynchronized lyrics](https://github.com/abhimanyuPathania/lyrico/issues/3).

See also my request for a [bash script downloader](https://github.com/osdlyrics/osdlyrics/issues/82).

**Comment by PedroHLC** [comment]
> > i recently installed lyrico with pip in virtualenv, it uses
> 
> [only supports unsynchronized lyrics](https://github.com/abhimanyuPathania/lyrico/issues/3).
> 
> See also my request for a [bash script downloader](https://github.com/osdlyrics/osdlyrics/issues/82).

Considering both projects uses interfaces in python, would be cool if @abhimanyuPathania adopted the few sources we had, make lyrco "library-compatible" and then we could re-use it as a dependency.

**Comment by dasj19** [comment]
So I'll join to this discussion party, nice lyric sources.
I have found another one: https://www.rentanadviser.com/en/subtitles/subtitles4songs.aspx and made a basic module for it here: https://github.com/osdlyrics/osdlyrics/pull/91

**Comment by dasj19** [comment]
Another potential source could be: https://www.megalobiz.com/search/all?qry=Above+Beyond
